views_api.py

```diff
@@ -130,6 +129,12 @@ async def lndhub_gettxs(
     ):
         await payment.check_status()
 
+    def extract_memo(payment):
+        try:
```

Comment:
- `try-except` should not be used for logic control
- use `if `comment in payment.extra: ...` instead